src/components/Sections/SectionDate.js

```diff
@@ -3,14 +3,40 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import moment from 'moment-timezone';
 
+const PARSING_STRING_WITH_TIME_ZONE = 'YYYY-MM-DD HH:mm:ss.SSSSSS Z';
+const DEFAULT_FORMAT = 'MMMM Do YYYY, h:mm:ss a Z';
 
-const SectionDate = ({ date, style, format }) =>
-  <div className="section-date" style={style}>
-    <span className="section-date-key">Date:</span>
-    <span className="section-date-value">
-      {date ? moment(date).tz(moment.tz.guess()).format(format) : moment().tz(moment.tz.guess()).format(format)}
-    </span>
-  </div>;
+function dateToMoment(date) {
+  if (!date) {
+    return moment();
```

Comment:
wouldn't this show the `now` time?

![image](https://user-images.githubusercontent.com/10514677/143457738-57768686-8262-49d7-a1d2-65522e0e596c.png)?

Reply:
umm yes, if `date` is falsy it will return `now` time. same as it behave before my change

Reply:
I think that this somewhat problematic, are you sure that if date field is empty it will print the current time?

Reply:
I agree this is problematic... but this was the actual behavior before the change

Reply:
see https://github.com/demisto/sane-reports/pull/199/files?diff=split&w=1#diff-0db8250ddabb12676157fe8e51380f6fc06fd52b61cdb431bda59a78382e4375L11

```js
{date ? moment(date).tz(moment.tz.guess()).format(format) : moment().tz(moment.tz.guess()).format(format)}
```

if `date` is falsy, current time will be output

Reply:
suggestion: i can change it to `N/A` if falsy

Reply:
yeah I think its better  👍

Reply:
will do

Reply:
@asafshen Note: it will cause all reports dates to be `N/A` on creation date. i see we never send the date from the web-client for title:date key

Reply:
gotcha, this is indeed problematic

lets leave it as is now and fix it later as a separate issue

Reply:
so on a second thought i'm not sure its a smart idea

Reply:
right, I agree 👍 

I think that an   ideal solution would be to create the current time on server, and then if date field has empty  value it will be N/A but creation will be ok

Reply:
@asafshen I had tried that (replace `now` with `N/A`) and it breaks the current behavior and tests. I had revert that

Reply:
👍 
lmk when to re-review